cloudwrapper/gcs.py

```diff
@@ -4,7 +4,11 @@
 Author: Martin Mikita <martin.mikita@klokantech.com>
 """
 
+import base64
+import crc32c
```

Comment:
move this import lower, between `try + raise ImportError` and add name of the package for installing via PIP

Reply:
Done.